### PR TITLE
Fix npm plugin for scopes, fail-safe mdn plugin, add utils/exec

### PR DESCRIPTION
**Pull request by caub**

### CODE REVIEW

**Comment by caub** [comment]
@ljharb can we skip the tests on node14? I used AbortController which is node16+

**Comment by ljharb** [comment]
I would strongly prefer not to use AbortController at all, unrelated to node versions.

**Comment by caub** [comment]
ok done, reverted without AbortController

**Comment by caub** [comment]
I wonder how the tests passed before because AbortController was already used in run.js https://github.com/ljharb/jellobot/blob/main/src/plugins/js-eval/run.js#L81 before this PR

**Comment by caub** [comment]
oh nvm, that part is not covered, and run in docker anyway

File: src/plugins/npm/npmPlugin.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+it('works for searches', async () => {
+  const output = await testNpm('npm ~bootstrap');
+
+  const results = output.split('\n');
+  expect(results.length).toBeGreaterThan(2);
+
+  expect(results[0].split('|')[0]).toEqual('npm.im/bootstrap');
+  expect(results[0].split('|')[1]).toMatch(/^\d+\.\d+\.\d+/);
+  expect(results[0].split('|')[2]).toMatch(/^\d{4}-\d{2}-\d{2}$/);
+});
```
Comment (ljharb) [inline src/plugins/npm/npmPlugin.test.js:31]: please ensure all files have trailing newlines

**Review by ljharb — COMMENTED** [review]
Turns out we weren't running prettier in the main eslint setup; that's fixed if you rebase.

File: src/plugins/npm/npmPlugin.test.js (hunk 1 above)
Comment (caub) [reply src/plugins/npm/npmPlugin.test.js]: Done, but we could maybe have done those things using eslint rule eol-last, no-trailing-spaces, comma and without the terrible prettier plugin

Comment (ljharb) [reply src/plugins/npm/npmPlugin.test.js]: i totally agree, and would be fine to remove prettier later - it was just using the prettier config already which disabled those rules, so i went with that.

**Review by ljharb — APPROVED** [review]

**Pull request merged**